### PR TITLE
Issue #5387: removed @return JavaDoc block tag from NonEmptyAtclauseD…

### PR DESCRIPTION
**Pull request by sikhlev**
Fix for Issue #5387

### CODE REVIEW

**Comment by romani** [comment]
@sikhlev , before pushing to PullRequest, please verify that on your local `mvn clean verify` pass.
Please review CI failures and fix them.

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputNonEmptyAtclauseDescriptionCheck.java [hunk 1]
```diff
@@ -7,7 +7,6 @@
 	 * @param a Some javadoc
 	 * @param b Some javadoc
 	 * @param c Some javadoc
-	 * @return Some javadoc
```
Comment (rnveach) [inline src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputNonEmptyAtclauseDescriptionCheck.java]: Why do inputs need to change?  Nothing hurts from leaving them in since check won't process them.
Plrease revert this change. This will fix the changes to line numbers too in the test.

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputNonEmptyAtclauseDescriptionCheck.java [hunk 2]
```diff
@@ -21,7 +20,6 @@ public int foo1(String a, int b, double c) throws Exception
...
```
Comment (rnveach) [inline src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputNonEmptyAtclauseDescriptionCheck.java]: Same.

**Comment by rnveach** [comment]
@sikhlev ping!

**Comment by romani** [comment]
@rnveach , please look at my comment in issue.

**Comment by romani** [comment]
after long discussion in issue page we decided to remove approved label from issue, it is better to keep behavior as is.
This PR is rejected, sorry for confusion.
